Sitka-Sound-Science-Center/sitka-landslide
# Add temporary banner for beta test

## Overview

Adds a temporary banner across header of application to indicate beta testing.

Resolves #22

### Demo

<img width="374" alt="Screen Shot 2022-05-03 at 3 59 07 PM" src="https://user-images.githubusercontent.com/1809908/166556130-feb13784-7fff-4d52-b861-5859414d9cfc.png">


### Notes

This should be removed after site goes live.

## Testing Instructions

 * Open application
 * Go to any page
 * You should see a banner across the top of the page that looks similar to the above screenshot
 * Try to close the banner; it should go away for this session.

## Checklist
